### DISCUSSION TITLE
Change for flamers

### DISCUSSION THREAD

**Discussion by element109** [ReactiveDrop/reactivedrop_public_src#193] [Ideas]
Require medic class to equip a flamer from the lobby, but any class can carry/use them in game.

**Comment by Duke-M-commits** [comment]
Nah, I don't get how this could be helpful. 

On another note: I would vote to drop the flamer requirement in the whole.
Needing a flamer for a mission can be solved by the mapper who's creating such a requirement in the first place. 
They sure can put some kind of helper (like a flamer lying around or a machine producing heat) in there.

**Reply by element109** [reply]
It is helpful by limiting the number of flamers being brought onto the map. If you ever joined a public game you would see the common occurrence of every player taking a flamer, which on any difficulty above easy is likely to cause failure. It makes it harder to aim because you cannot see as well with flames everywhere. It causes performance issues on lower spec machines. It causes griefing and the always annoying marine on fire screaming. It is just bad game play period.

**Reply by Duke-M-commits** [reply]
Oh, i didn‘t realize you mean 2 flamers by medics maximum.

**Reply by element109** [reply]
Right, so from the lobby each medic can only take one flamer. That would mean there would be only two flamers in game, unless there is some on the map or spawned by a challenge.

**Reply by ax-now** [reply]
So you want to solve a problem of performance and bad atitude of players by reducing the possibility of gameplay? meh

I think it's a bad idea, the flamer is a newbie friendly weapon and I don't see what the problem is if it is used in low difficulties because it is very powerful and pleasant to play when you start the game. I don't know where you played your difficult games but for my part I see very often flamer because it is not a very effective weapon at high difficulty. 
So it's true, it happens that a newbie joins difficult games with flamer and burns players, but it's very rare and in general it is enough to explain to him or to ask for a kick if he does not want to understand.
flamer hate is a noob problem imo, I myself had trouble with this kind of players at the beginning but now it does not matter anymore, the flamer is not a real problem at high difficulty and it does not do so much friendly fire at low difficulty.

**Reply by Duke-M-commits** [reply]
What you are basically saying is, you are over it and now you do not see anymore where it is.
But everyone is a noob at some point in life when he starts something.
The flamer issue is a problem in the introduction of the game.
A good tutorial needs some love too.

**Reply by ax-now** [reply]
Its not only about flamer, gameplay restriction is bad thing, this also applies to sentry and I heard that he also wants to apply this restriction to the grenade launcher.
It would be more interesting to think differently, add more gameplay variety rather than reduce it.
It's my simple player opinion maybe I'm wrong but I think a lot of other players will agree with me.

**Reply by Duke-M-commits** [reply]
But you realize the problem exists don't you? Any better ideas maybe? I mean you have a lot of experience as far as I know, and you for sure did talk about it more then him or me over time. The only thing I read over time is, no no no, don't change anything, and this doesn't bring anything forward either. Making some changes doesn't mean they are written in stone.

Especially if such stuff can stay in the beta, I mean I'm pretty sure Ben can leave some stuff in the beta forever while still bringing regular updates to the main version.

**Reply by ax-now** [reply]
there may be a "flamer problem" among casual players, but the restriction is not the right solution, penalizing the entire player base for a problem that only affects a small part of the players. even if its 50% of player base.

I understand that you want to do well for the game but you should ask for advice and good players who know the game inside out and luckily you had some who are even part of the dev group but you also had who are simple players, there is a communication problem, offer you major change ideas on a small development group then pass them a beta without even communicating with people who know this game well. 
Ha so sorry Element left a message on the steamrd chat a discussion file that no one reads :(, as much as leaving a message on facebook it will have the same effect. 
I don't know, you could maybe communicate on the steam forums (I mean before you put your idea in beta) to see if it's a good idea or not.
So of course there will be very irrelevant opinions (I'm not asking you to listen to everyone) but it would allow you to have a global point of view on the opinion of the community. 
This is not only true for the flamer changes but for any major change that changed the game enormously which could divide opinions.

Anyway, that my opinion will be taken without aggressiveness, it is only the opinion of a player in love with the game for years and who knows it enough to see if a change could pose a problem.

Thanks for your time and stay safe.

**Comment by element109** [comment]
It is so bad there is a meme about it.

**Comment by element109** [comment]
Cheat convars would be created to disable the limit but would be enabled by default.

**Comment by Duke-M-commits** [comment]
I don‘t know, I heard of much ideas how to tackle that problem and this, like other ideas sounds like fighting the symptom and not the root of the problem. I think the flamer should be nerved, like a lot. How about a really short mag? Like 10, and a spare mag with 10. So it behaves like a secondary weapon.

**Comment by Duke-M-commits** [comment]
Or maybe throw the flamer out in the whole (from the loadout screen, could be still lying around in the level) and make it a secondary fire mode on the heal/amp gun (i have the feeling nobody ever uses the amp of the medgun).

**Reply by Duke-M-commits** [reply]
Such a change would eliminate this problem in the whole. Nobody takes a flamer initially, Officers and Medics can play with fire just a bit and that’s it, no hell-bent flamer guys anymore, no flamer requirement anymore. This could be rounded up with the welder being able to ignite biomass …

**Comment by Bountyover** [comment]
The problem I see with this solution is people will pick med instantly just to play with flamer. So the chance of having a good medic will be much lower.

Also, people who play with flamer are usually not competent enough to deal with biomass. And relying on Officer's fire nades is not always an option.

**Reply by Duke-M-commits** [reply]
There are multiple ideas in here, you should reply to the idea you are referring to.

**Reply by element109** [reply]
Another option would be to remove the medic class requirement, and just make it so any class can take the flamer but still limit it to two. So once two marines have equipped the flamer it would no longer be available to equip until one of the marines unequipped it. We would need to make sure one marine does not equip both of them as well. There would need to be a message shown to players telling them when the two flamer limit has been reached and that is why they are unable to equip it.

**Reply by Duke-M-commits** [reply]
That sounds way to complicated for a barely intuitive UX sorry.

**Reply by Bountyover** [reply]
> Another option would be to remove the medic class requirement, and just make it so any class can take the flamer but still limit it to two. So once two marines have equipped the flamer it would no longer be available to equip until one of the marines unequipped it. We would need to make sure one marine does not equip both of them as well. There would need to be a message shown to players telling them when the two flamer limit has been reached and that is why they are unable to equip it.

This would be much better but also harder to do.

**Reply by element109** [reply]
Can Vegas be "equipped" by 2 different players?

**Reply by BenLubar** [reply]
Adding a way for challenges to limit the number of a specific weapon seems doable.

**Reply by mithrand0** [reply]
Lore wise, flamers should be heavy, it's a heavy weapon.

If the flamer would slow down the marine, similar like the autogun, it would be not useful anymore for rushing, while maintaining it's ability to grill parasites and cover the back of a team. Slightly limit the amount of ammo per clip and rebalancing should be done.

**Reply by BenLubar** [reply]
What speed multiplier do we give to flamer?

Most guns (including flamer currently): 50% firing, 100% idle
Autogun: 50% firing, 80% idle
Devastator: 30% firing, 70% idle
Minigun: 40% firing or spinning, 95% idle
Railgun: 75% firing, 100% idle
Sniper: 55% firing or zoomed, 100% idle
Generic Object: mapper-defined

**Reply by mithrand0** [reply]
I would say Devastator or Autogun level slowdown, but it might be good to hear con's and pro's from community members here.

Opened a discussion https://steamcommunity.com/app/563560/discussions/2/3361397532258307150/ to get some input on this topic.

**Comment by mithrand0** [comment]
This topic is quite old, and no balance changes are planned for the flamer right now. Closing it.
